lyatools/quickquasars.py

```diff
@@ -154,12 +154,13 @@ def create_qq_script(qq_tree, config, job, qq_args, qq_seed):
     return script_path
 
 
-def make_catalogs(qq_tree, config, job, dla_flag, bal_flag, qq_job_id, run_local=True):
+def make_catalogs(qq_tree, config, job, dla_flag, bal_flag, qq_job_id, only_qso_targets, run_local=True):
     command = ''
 
     zcat_file = qq_tree.qq_dir / 'zcat.fits'
     if not zcat_file.is_file():
-        command += f'lyatools-make-zcat -i {qq_tree.spectra_dir} -o {zcat_file} --nproc {128}\n\n'
+        only_qso_targets_flag = "--only_qso_targets" if only_qso_targets else ""
+        command += f'lyatools-make-zcat -i {qq_tree.spectra_dir} -o {zcat_file} --nproc {128} {only_qso_targets_flag}\n\n'

```

Comment:
The default zcat file name is still used, so it won't run the new catalog if the baseline one already exists. I think this could be fixed with something along the lines of:
```python 
if only_qso_targets:
    zcat_file = qq_tree.qq_dir / 'zcat_only_qso_targets.fits'
else:
    zcat_file = qq_tree.qq_dir / 'zcat.fits'
```

Reply:
Then the rest of the code in this if statement `if not zcat_file.is_file():`  would work as is now